openapi/doc/index.rst

```diff
@@ -21,43 +21,49 @@ Authentication
 
 * Open menu ``[[ Settings ]] >> Users >> Users``
 * Select a user that will be used for iteracting over API
-* In ``REST API`` tab Click  ``[Generate Access Token]``
+* Next to ``REST API`` field, click  ``[Generate Access Token]``
+* Next to ``REST API`` field, specify token expiry
 * Copy **Basic Authentication code** to use in any system that support REST API (Open API)
-
+* If you want to refresh the token progrmatically from the connected system, also copy the **Refresh Token**.
 
 Activating and customization
 ----------------------------
 
-via Model's Menu (recommended)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+directly on the Model
+~~~~~~~~~~~~~~~~~~~~~
 
 TODO
-
-* Open menu with data you need to work over API
-* Click button ``[Configure API]`` located near to ``Create
-* Activate and configure operation you need to be available over API.  
+* `Activate Developer Mode <https://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html>`__
+* Open the developer tools drop down
+* Click menu ``Configure REST API`` located within the dropdown
+* On the form that opens, activate and configure this module for REST API accessability. 
 * Click ``[Apply]``
 
-via Database Structure Menu (only for developers)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
+via Database Structure Menu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * `Activate Developer Mode <https://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html>`__
 * Open menu ``[[ Settings ]] >> Technical >> Database Structure >> Models``
-* Open model you need
+* Open the model you want to configure
 * Switch to ``REST API`` tab
-* Set some of operations and configuration for them:
+* On that tab, activate and configure this module for REST API accessability. 
+
+
+REST API Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+* Enable CRUD operations individually
+* Set configuration for ReadOne, ReadMulti and optionally Create operations:
```

Comment:
ReadOne and ReadMulti are only for Read opearions, so could you move it under ``**[x] Read via API**`` section. The same about ``Create``